redis/src/main/scala/zio/redis/options/Cluster.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2021 John A. De Goes and the ZIO contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package zio.redis.options
+
+import zio.redis.{RedisExecutor, RedisUri}
+import zio.{Chunk, Scope}
+
+trait Cluster {}
```

Comment:
Is this marker necessary?

Reply:
No, Just for consistency with other) 
Drop it.